profiler/static/description/index.html

```diff
@@ -0,0 +1,53 @@
+<head>
```

Comment:
This module will be a OCA one
Then we could delete this index and Vauxoo's icon